tasks/preflight.yml

```diff
@@ -15,3 +15,23 @@
     msg: "This benchmark is not suitable for the destination operating system"
   when: (ansible_distribution is not defined) or (ansible_distribution_version is not defined) or (ansible_distribution_version not in cis_target_os_versions) or (ansible_distribution != cis_target_os_distribution)
   tags: always
+
+- name: Running in Vagrant?
```

Comment:
Wouldn't running in VM make more sense? This could be using Vagrant or running in EC2.

Reply:
Addressed in b881da6c4ada61ec08b4145aa1e1fe2e3cb933e2